app/submitter/converter.py

```diff
@@ -76,6 +77,13 @@ def convert_answers(schema, questionnaire_store, routing_path, flushed=False):
         'questionnaire_id': metadata['questionnaire_id'],
     }
 
+    data = {
```

Comment:
Any reason this is outside the 0.0.3 if statement, I can't see if being used anywhere else?

Reply:
It's done now